employees.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Employees</title>
+    <link rel="stylesheet" href="./css/employees.css">
+</head>
+<body>
+    <section class="left-navigation">
```

Comment:
we should prefer to use the semantic tags whenever possible.
like `aside` tag for sidenav